src/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+const STATUS_CHECKED = 'STATUS_CHECKED';
```

Comment:
* Same thing here Kindly use the ducks' pattern for your action types value as its a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Write%20your%20book%27s%20actions%20and%20reducer%20(in%20/src/redux/books/books.js)%20using%20the%20ducks%20pattern.)